pkg/controller/admissionchecks/multikueue/multikueuecluster.go

```diff
@@ -845,3 +860,47 @@ func (s *secretHandler) queue(ctx context.Context, secret *corev1.Secret, q work
 	}
 	return nil
 }
+
+type clusterProfileHandler struct {
+	client client.Client
+}
+
+var _ handler.EventHandler = (*clusterProfileHandler)(nil)
+
+func (cp *clusterProfileHandler) Create(ctx context.Context, event event.CreateEvent, q workqueue.TypedRateLimitingInterface[reconcile.Request]) {
+	cp.handleEvent(ctx, event.Object, q)
+}
+
+func (cp *clusterProfileHandler) Update(ctx context.Context, event event.UpdateEvent, q workqueue.TypedRateLimitingInterface[reconcile.Request]) {
+	cp.handleEvent(ctx, event.ObjectNew, q)
+}
+
+func (cp *clusterProfileHandler) Delete(ctx context.Context, event event.DeleteEvent, q workqueue.TypedRateLimitingInterface[reconcile.Request]) {
+	cp.handleEvent(ctx, event.Object, q)
+}
+
+func (cp *clusterProfileHandler) Generic(ctx context.Context, event event.GenericEvent, q workqueue.TypedRateLimitingInterface[reconcile.Request]) {
+}
+
+func (cp *clusterProfileHandler) handleEvent(ctx context.Context, object client.Object, q workqueue.TypedRateLimitingInterface[reconcile.Request]) {
+	clusterprofile, isClusterProfile := object.(*inventoryv1alpha1.ClusterProfile)
+	if !isClusterProfile {
+		ctrl.LoggerFrom(ctx).V(5).Error(errors.New("not a clusterprofile"), "Failure on event")
+		return
+	}
+
+	mkcList := &kueue.MultiKueueClusterList{}
+	if err := cp.client.List(ctx, mkcList, client.MatchingFields{UsingClusterProfiles: strings.Join([]string{clusterprofile.Namespace, clusterprofile.Name}, "/")}); err != nil {
+		ctrl.LoggerFrom(ctx).V(5).Error(err, "Failure on event", "clusterprofile", klog.KObj(object))
```

Comment:
```suggestion
		ctrl.LoggerFrom(ctx).Error(err, "Failure on event", "clusterprofile", klog.KObj(object))
```
Errors should be v(5)